src/quo2/components/lowest_price.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.lowest-price
+  (:require [quo.design-system.colors :as colors]
```

Comment:
quo2 colors should be used

Reply:
https://github.com/status-im/status-react/blob/8e29231a88cc0771221a7e7c4306191d457a7239/src/quo2/screens/main.cljs#L3-L4

It's used inside quo2 to get the theme value if it's dark/white
Should we remake a theming namespace for quo2?

Reply:
theming should be done in separate ns

Reply:
Please check it now